packages/slate-react/src/plugin/with-react.ts

```diff
@@ -69,12 +69,12 @@ export const withReact = <T extends Editor>(editor: T) => {
     const text = data.getData('text/plain')
 
     if (text) {
-      const lines = text.split('\n')
+      const lines = text.split(/\r\n|\r|\n/g)
```

Comment:
The use of the g here would make sense if we're trying to get multiple matches for our pattern string, but in the context of split, I'm guessing we are only interested in whether there's one match on the remaining untokenized text. The g might be making this regex do more work than necessary if it's trying to match all instances of /\r\n|\r|\n/ through the whole string for every token? 

Interested in your thoughts on whether the g is necessary, can be dropped, or whether it's negligible either way.

Reply:
Yep, I think you're right that it's unnecessary. I tested the example without `/g` and it still works. I updated the PR to remove it.